### ISSUE TITLE
A change of font & its size not well applied to server list and user list

### ISSUE THREAD

**Issue by BBasile** [hexchat/hexchat#2023]
Under Windows 7, there's something wrong with the height of the items, let's say if you go from 10 to 15, the user list and the server list items height is not updated but the text has the good size.

**Comment by learn-more** [comment]
I would assume that the item height is calculated once, and not properly re-calculated when font size changes.

**Comment by TingPing** [comment]
Just a guess but maybe its caused because we set it as fixed height: https://github.com/hexchat/hexchat/blob/15600f405f2d5bda6ccf0dd73957395716e0d4d3/src/fe-gtk/chanview-tree.c#L156

**Comment by TingPing** [comment]
@BBasile This issue is not solved, please stop closing it.

**Comment by Martchus** [comment]
Yes, it isn't solved. And the issue is also not limited to Windows. Since the issue might be good for a first commit, so I'm going to look into it :-)
